### PR TITLE
Add one line text mixin to common styles

### PR DESCRIPTION
**Pull request by jalack**

### CODE REVIEW

File: lib/stylesheets/mixins/common.styl [hunk 1]
```diff
@@ -13,6 +13,11 @@ z-dropdown = 7000
 mp-box-shadow(size=10)
   box-shadow 0 (size)px (size * 4)px rgba(0, 0, 0, 0.15)
 
+one-line-text()
+  text-overflow ellipsis
+  white-space nowrap
+  overflow hidden
```
Comment (tdumitrescu) [inline lib/stylesheets/mixins/common.styl]: styles a-z (didn't @jbwyme add a stylus linter for this?)

Comment (tdumitrescu) [inline lib/stylesheets/mixins/common.styl]: might this be better in https://github.com/mixpanel/mixpanel-common/blob/36483bbcb087225aa01d5daa1762de4f098668e5/lib/stylesheets/mixins/typography.styl?

Comment (jbwyme) [inline lib/stylesheets/mixins/common.styl]: re: linter, there is one but turns out it's just throwing warnings for everything so I've got a branch in waiting that turns them all to errors

Comment (jbwyme) [inline lib/stylesheets/mixins/common.styl]: also i don't think we should use this mixin since we want to truncate middle everywhere

Comment (jalack) [inline lib/stylesheets/mixins/common.styl]: good to know!

**Pull request closed without merging**